src/cephadm/cephadm

```diff
@@ -6500,9 +6499,10 @@ class CephadmDaemon():
     loop_delay = 1
     thread_check_interval = 5
 
-    def __init__(self, ctx: CephadmContext, fsid, daemon_id=None, port=None):
+    def __init__(self, ctx: CephadmContext, fsid, binary, daemon_id=None, port=None):
         self.ctx = ctx
         self.fsid = fsid
+        self.binary_path = binary
```

Comment:
maybe it would be nice to have a verification  to know that in the patch provided there is "something".

Reply:
done